apps/web/src/pages/Users/AdminUserProfilePage/components/UserProfilePrintButton.tsx

```diff
@@ -1,46 +1,106 @@
-import React, { useRef } from "react";
+import React, { useRef, useEffect, useState } from "react";
 import { useIntl } from "react-intl";
 import { useReactToPrint } from "react-to-print";
+import ChevronDownIcon from "@heroicons/react/24/solid/ChevronDownIcon";
 
-import { Button } from "@gc-digital-talent/ui";
+import {
+  Button,
+  ButtonLinkMode,
+  Color,
+  DropdownMenu,
+} from "@gc-digital-talent/ui";
 
-import { User } from "~/api/generated";
+import { PoolCandidate, User } from "~/api/generated";
 import printStyles from "~/styles/printStyles";
 import ProfileDocument from "~/components/ProfileDocument/ProfileDocument";
 
 interface UserProfilePrintButtonProps {
-  user: User;
-  children?: React.ReactNode;
-  anonymous?: boolean;
+  users: User[] | PoolCandidate[];
+  color: Color;
+  mode: ButtonLinkMode;
+  beforePrint?: (handlePrint: () => void) => void;
 }
 
 const UserProfilePrintButton = ({
-  user,
-  children,
-  anonymous,
+  users,
+  color,
+  mode,
+  beforePrint,
 }: UserProfilePrintButtonProps) => {
   const intl = useIntl();
 
+  const [isAnonymous, setAnonymous] = useState("");
```

Comment:
Could you constrain the type on this?  Probably a string union?

Reply:
Sure thing 👍 Fixed by [8f8c9c7](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7749/commits/8f8c9c7aa60474fc9c5213646d708f5e0cc4e7c1)